scripts/report_scripts/index.Rmd

```diff
@@ -1,6 +1,6 @@
 ---
-title: "Timecourse"
-date: '`r format(as.POSIXct(if ("" != Sys.getenv("SOURCE_DATE_EPOCH")) { as.numeric(Sys.getenv("SOURCE_DATE_EPOCH")) } else { Sys.time() }, origin="1970-01-01"), "%Y-%m-%d %H:%M:%S")`'
+title: "SARS-CoV-2 Variants of Concern - Timecourse report"
+date: '`r report generated: format(as.POSIXct(if ("" != Sys.getenv("SOURCE_DATE_EPOCH")) { as.numeric(Sys.getenv("SOURCE_DATE_EPOCH")) } else { Sys.time() }, origin="1970-01-01"), "%Y-%m-%d %H:%M:%S")`'
```

Comment:
For some reason, in our case, I found it very confusing. Confusing in terms of whether it's the timestamp of the report or the sample date. From the Rna seq reports, however, I don't get this feeling. Maybe it's because of the additional name....
but also, maybe it's just unnecessary rambling

Reply:
Yeah, it doesn't look good, but I think we shouldn't work around this by overloading the `date` field in the yaml front matter.  The front matter has certain semantics that I wouldn't like to conflate with what is essentially a styling fix.